src/config/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
+filters:
+  "(OWNERS|mod\\.rs)$":
+    approvers:
+      - sig-critical-approvers-config-transaction
```

Comment:
@overvenus Which module is this file primarily concerned with?

Reply:
All modules, as it is the actual TiKV configuration struct.